modules/core/src/main/scala/dev/profunktor/redis4cats/connection/RedisClient.scala

```diff
@@ -140,13 +141,14 @@ object RedisClient {
         opts: ClientOptions,
         config: Redis4CatsConfig = Redis4CatsConfig()
     ): Resource[F, RedisClient] =
-      RedisExecutor.make[F].flatMap { implicit redisExecutor =>
+      MkRedis[F].newExecutor.flatMap { implicit ec =>
         val (acquire, release) = acquireAndRelease(uri, opts, config)
         Resource.make(acquire)(release)
       }
   }
 
-  def apply[F[_]: Async: Log]: RedisClientPartiallyApplied[F] = new RedisClientPartiallyApplied[F]
+  def apply[F[_]: FutureLift: Log: MkRedis: MonadThrow]: RedisClientPartiallyApplied[F] =
```

Comment:
Is the `FutureLift` and `Log` needed here, given MkRedis provides them?

Reply:
They are not, good catch. Good opportunity to let the CI build run multiple times :)